### PR TITLE
[importer] increasing the value of DATA_UPLOAD_MAX_MEMORY_SIZE to 5MB  as sometime request size > 2.5MB (default value)

### PR DESCRIPTION
**Pull request by agl29**
## What changes were proposed in this pull request?

-  Increasing the value of DATA_UPLOAD_MAX_MEMORY_SIZE to 5MB.

## How was this patch tested?

- Manually.

### CODE REVIEW

File: desktop/core/src/desktop/settings.py [hunk 1]
```diff
@@ -117,6 +118,8 @@
 # Examples: "http://media.lawrence.com/media/", "http://example.com/media/"
 MEDIA_URL = ''
 
+if ENABLE_DIRECT_UPLOAD:
```
Comment (Harshg999) [inline desktop/core/src/desktop/settings.py:121]: Shouldn't it be `ENABLE_DIRECT_UPLOAD.get()`?

Comment (Harshg999) [reply desktop/core/src/desktop/settings.py:121]: This doesn't look like a `generic configuration`, I think we should make a separate section for Indexer like at the end for other specific configs and put it there.

Comment (agl29) [reply desktop/core/src/desktop/settings.py]: 1. Thanks @Harshg999.

2. This variable comes under the core settings that's why I putted that variable in  generic configuration. any thoughts @ranade1 @wing2fly ?

**Review by JohanAhlen — APPROVED** [review]

**Review by ranade1 — APPROVED** [review]
looks good to me.

**Pull request merged**